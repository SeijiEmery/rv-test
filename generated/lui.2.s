lui a1, 0xEF51A
addi zero, zero, 0
addi zero, zero, 0
addi zero, zero, 0
addi zero, zero, 0
addi zero, zero, 0
ebreak
