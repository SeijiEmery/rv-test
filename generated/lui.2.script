load /x 65444 generated/lui.2.hex
writereg -1 65444
writereg 11 0
setpc 65444
run 5
readreg -1
readreg 11