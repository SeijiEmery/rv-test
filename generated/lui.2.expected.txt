tests/lui.test.s:14
pc = 65448
R11 = 18446744073429688320
